clash/bloody_grail.tex

```diff
@@ -9,122 +9,115 @@
 
 \textit{A Lady clad in dark robes visits you in your dreams, offering you a golden chalice full of blood-red wine. Waking up in a cold sweat, you decide to embark on a quest to find this unholy artifact.}
 
-\subsection*{\MakeUppercase{Player Count}}
-A scenario for 2 or 4 players.
-
 \subsection*{\MakeUppercase{Scenario Length}}
-This scenario plays out over 13 rounds.
+This scenario is played over 13 rounds.
+
+\subsection*{\MakeUppercase{Player Setup}}
+\textbf{Player Count:} 2 or 4
+
+\textbf{Starting Resources:}\par
+\resources{13}{2}{1}
+
+\textbf{Starting Income:}\par
+\resources{10}{0}{0}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item A Few \svg{bronze} units with the \textit{highest} Recruitment cost
+  \item A Pack of \svg{bronze} units with the \textit{lowest} Recruitment cost
+\end{itemize}
+
+\textbf{Town Buildings:} \svg{bronze} Dwelling
+
+\textbf{Map Tile Pool:} Each player takes 2 random Far (II-III) Map Tiles, one of which must contain a Settlement.
+
+\textbf{Starting Bonus:} None
```

Comment:
This can be confusing and lead people into erroneously thinking this section overrides the default starting bonus due to Difficulty, which is not the intent. For scenarios where the players take the default Difficulty bonus, I'd omit this section, or rename it to "Additional Starting Bonus:" for example.

Reply:
Argh, sorry, I thought it was supposed to be an actual replacement. I'll fix it.